chat/chat-disclaimer/src/index.ts

```diff
@@ -1,2 +1,5 @@
+/**
+ * @deprecated - use `@leafygreen-ui/marketing-modal` instead
```

Comment:
Is there also a component that we can add this to?

Reply:
When you say, remove the export, do you mean delete that package?

Reply:
Should we keep it and archive it? I remember we spoke about archiving deprecated components in dev sync. However, this is such a tiny component, it probably doesn't make sense to archive it.